packages/flutter/lib/src/material/ink_well.dart

```diff
@@ -1458,7 +1458,7 @@ class InkWell extends InkResponse {
     super.radius,
     super.borderRadius,
     super.customBorder,
-    bool? enableFeedback = true,
+    super.enableFeedback,
```

Comment:
Technically this could be a breaking change for someone, if they've explicitly called `InkWell(enableFeedback: null)`. I do think this is a nice readability improvement though.

Reply:
I’m not entirely sure how this should be adjusted. Should I mark this PR as a breaking change? It doesn't seem like it's significant enough for that, but I'm unsure. Could you provide some guidance? Thank you!